CONTRIBUTING.md

```diff
@@ -80,7 +80,7 @@ Just like php dependencies can be managed with Composer, python dependencies can
 To get sphinx, simply run the following command.
 
 ```bash
-pip install --requirement Resources/doc/requirements.txt --user
+pip install --requirement docs/requirements.txt --user
```

Comment:
This file is managed through dev-kit. Maybe we should recommend `make docs` instead?

Reply:
We already recommend `make docs`, if you read the whole paragraph.

Reply:
Oh yeah I'm tired, that's not about producing the docs, it's about installing sphinx. Sorry for the trouble.